webapp/TargetedMS/js/QCPlotHelperBase.js

```diff
@@ -175,6 +178,27 @@ Ext4.define("LABKEY.targetedms.QCPlotHelperBase", {
                 allPlotDateValues.push(data.fullDate);
 
             }, this);
+
+            if (this.filterQCPoints && !this.groupedX) {
+                for (var j = 0; j < plotDataRow.data.length; j++) {
+                    var plotData = plotDataRow.data[j];
+
+                    // for truncating out of range guideset data find last index of plotData starting from this.startDate
+                    if (plotData.AcquiredTime >= this.startDate) {
+                        if (!this.outOfRangeGSLastIndex) {
+                            // skip 5 points
+                            this.outOfRangeGSLastIndex = j - 5;
```

Comment:
thanks for catching, it was always putting a break but now I have modified it so that graph will be continuous if the points between end of reference guideset and custom start date are less than 6.